README.md

```diff
@@ -111,18 +134,20 @@ The **#matrix** project has some environments that important to define.
 
 ### External Meet
 The default video conferencing in meetings is [Jitsi](https://jitsi.org/jitsi-meet/), but you can change that in any room, using [Meet](https://meet.google.com/) or [Zoom](https://zoom.us/). For that, you just need provide the parameter `externalMeetUrl` in your room config:
-```
-ROOMS_DATA=[
+
+		ROOMS_DATA=[
 		   {
 		      "id":"${UUID}",
 		      "name":"Meeting External",
 			  "externalMeetUrl": "https://external-url-room/key-room"
 		   }
 		 ]
-```
+
 
 # Running in Production
-If you will run in production we strongly recomend to you close your environmen using an internal VPN. In this solution everybody with the link and a valid google credential can enter in your virtual office. Because of this is important to you mantain your environment closed. Or you can help us implementing to define a whitelist domains with environment variable. 
+If you will run in production we strongly recomend to you close your environmen using an internal VPN. In this solution everybody with the link and a valid google credential can enter in your virtual office. Because of this is important to you mantain your environment closed. Or you can define a variable `WHITELIST_DOMAINS` to limit only authorized users enter in the **#matrix**. 
```

Comment:
```suggestion
If you will run in production we strongly recommend you close your environment using an internal VPN. In this solution everybody with the link and a valid google credential can enter your virtual office. Because this is important for you to maintain your environment closed. Or you can define a variable `WHITELIST_DOMAINS` to limit only authorized users to enter in the **#matrix**. You have to choose a strong key to the `COOKIE_SESSION_SECRET` and have to put in `GOOGLE_CALLBACK_URL` your production domain.
```